src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,31 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
```

Comment:
In this context you can use `! empty` as `isset` becomes redundant. 

```suggestion
		if ( 'scheduled' === $this->tab->get_slug() &&  ! empty( $_REQUEST['s'] ) ) {
```